tests/integration_tests/test_multi_single_AS.py

```diff
@@ -0,0 +1,225 @@
+from openmdao.utils.assert_utils import assert_near_equal
+import unittest
+
+
+class Test(unittest.TestCase):
```

Comment:
Again a brief comment would help, something like we compare multi-section and single-section for the same geometry and check we get the same results